examples/CustomComponent.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { List, arrayMove } from '../src/index';
+import { IItemProps } from '../src/types';
+
+const CustomItem = React.forwardRef(({ children, ...props }: IItemProps & {
+    children: string,
+}, ref: React.LegacyRef<HTMLLIElement>) => (
```

Comment:
hm, why `React.LegacyRef`?

Reply:
I'm not very experienced with TypeScript so I was just trying to make it shut up :) but it seems React.Ref also works and is less weird.